$OpenBSD: patch-src_bin_e_init_main_c,v 1.2 2014/03/26 08:55:26 sthen Exp $
hide splash screen after 5 seconds, for some reason it doesn't time out properly
--- src/bin/e_init_main.c.orig	Mon Jul 29 18:12:34 2013
+++ src/bin/e_init_main.c	Wed Mar 26 08:54:44 2014
@@ -344,7 +344,7 @@ e_init_init(void)
                                    _e_init_cb_signal_done_ok, NULL);
    free(roots);
 
-   _e_init_timeout_timer = ecore_timer_add(240.0, _e_init_cb_timeout, NULL);
+   _e_init_timeout_timer = ecore_timer_add(5.0, _e_init_cb_timeout, NULL);
    return 1;
 }
 
